### ISSUE TITLE
Bug with new versions of browsers?

### ISSUE THREAD

**Issue by jordi-domingo** [EIDA/webdc3#4]
Hi all,

I have detected the following error message in the console for all installations of WebDC3 (ODC, GFZ, ETH...):

`TypeError: a.children[d] is undefined` (Firefox)
`Uncaught TypeError: Cannot read property 'style' of undefined` (Chrome)

I saw it for the following browser versions...
Firefox 45.0, 46.0
Chrome 50.0, 51.0

Have you noticed it? I didn't see it in the past...
Something must have changed in the new versions of the browsers (or external libraries?), because it is independent on the WebDC3 version.

**Comment by pevans-gfz** [comment]
Yes, this appears to be related to our use of OpenLayers, which isn't compatible with a newer Google API. One workaround is for the installations (datacentres) to use a different base map: replace
js.maptype = google
by
js.maptype = wms
and accept (for now) the less-attractive maps.
Thanks for reporting it.

**Comment by jordi-domingo** [comment]
You're welcome.
I changed the configuration at ODC EIDA portal. The maps are less attractive, but not that bad. And the error doesn't show up now.

Thanks.
